tests/test_unclassified_job.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from unittestzero import Assert
+
+from selenium.webdriver.common.by import By
+
+from pages.resultset import ResultsetPage
+from pages.resultset import LogviewerPage
+
+
+class TestUnclassifiedJobs:
+
+    @pytest.mark.nondestructive
+    def test_unclassified_failure(self, mozwebqa):
+        # Open resultset page and search for next unclassified failure
+        resultset_page = ResultsetPage(mozwebqa)
+        resultset_page.go_to_page()
+        resultset_page.open_next_unclassified_failure()
+
+        teststatus = resultset_page.job_details_status
```

Comment:
Running locally I kept getting failures on both tests, but I put a print in here and it appears that this value is empty for some reason. I'll dig around more when I get the chance.